engine/admin/multi-process_container.md

```diff
@@ -0,0 +1,89 @@
+---
+description: How to run more than one process in a container
+keywords: docker, supervisor, process management
+redirect_from:
+- /engine/articles/using_supervisord/
+- /engine/admin/using_supervisord/
+title: Run multiple processes in a container
+---
+
+A container's running process is usually the `ENTRYPOINT` and/or `CMD` at the
+end of the `Dockerfile`. It is generally recommended that you separate areas of
+concern by using one container per process, and connect those containers using
+user-defined networks and shared volumes.
+
+If you need to run more than one process within a container, you can accomplish
+this in a few different ways:
+
+- Chain multiple commands together in the `CMD`:
+
+  ```conf
+  FROM ubuntu:latest
+  COPY my_first_process my_first_process
+  COPY my_second_process my_second_process
+  CMD ./my_first_process && ./my_second_process
```

Comment:
This probably won't work, because it runs the second process _after_ the first one exits; you probably have to daemonize the first one (`./foobar &`) to keep it running in the background.

I'm actually not sure we should recommend this, as this is a DYI init process, without reaping, and without any management of (e.g.) logging, startup ordering, keeping processes up, etc.

I'd prefer to keep the supervisord page as a _possible_ option to run multiple processes, for those that need it, as long as we reiterate that generally  running a single service in a container is the way to go.

Probably we can come up with some examples how to split those services over multiple containers (e.g. have a `sshd` service container that allows you to rsync files in a volume that's used by another container)